### DISCUSSION TITLE
TransferBuffs / TransferAfflictions applies all afflictions to mainlimb

### DISCUSSION THREAD

**Discussion by A2ur3** [FakeFishGames/Barotrauma#15285] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [X] My issue happened while using mods.

### What happened?

creature has 1.0 VitalityMultiplier on Torso, 0.5 VitalityMultiplier on legs
180% gunshot on left leg = 90% HP decrease
TransferAfflictions="true" and it will have 180% gunshot on Torso = 180% HP decrease

### Reproduction steps

.

### Bug prevalence

Just once

### Single player or multiplayer?

Happens in both single player and multiplayer

### Version

v1.6.19.1 (Unto the Breach Update Hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
This has been addressed in the latest unstable build, and the fix will be included in the next update.
